pkg/channel/event_receiver_test.go

```diff
@@ -121,6 +121,56 @@ func TestEventReceiver_ServeHTTP(t *testing.T) {
 			},
 			expected: http.StatusAccepted,
 		},
+		"host URL with scheme -- headers and body pass through": {
+			// The header, body, and host values set here are verified in the receiverFunc. Altering
+			// them here will require the same alteration in the receiverFunc.
+			header: map[string][]string{
+				"not":                       {"passed", "through"},
+				"nor":                       {"this-one"},
+				"x-requEst-id":              {"1234"},
+				"knatIve-will-pass-through": {"true", "always"},
+				// Ce headers won't pass through our header filtering as they should actually be set in the CloudEvent itself,
+				// as extensions. The SDK then sets them as as Ce- headers when sending them through HTTP.
+				"cE-not-pass-through": {"true"},
```

Comment:
CloudEvents v1 disallows this format.

Reply:
What part of this? It's just copied from the test case above, so perhaps an update to this repo was missed?

Reply:
I'm not especially familiar with this test, but in general we support older versions of CloudEvents in eventing components, so this seems like a reasonable test case to me.

Reply:
in 1.0 cloudevents does not allow the `-` char to be inside the key name.